llvm/include/llvm/MC/CAS/MCCASObjectV1.def

```diff
@@ -35,6 +37,8 @@ CASV1_SIMPLE_GROUP_REF(DebugAbbrevSectionRef, mc:debug_abbrev_section)
 CASV1_SIMPLE_GROUP_REF(DebugLineSectionRef, mc:debug_line_section)
 CASV1_SIMPLE_GROUP_REF(AtomRef, mc:atom)
 CASV1_SIMPLE_GROUP_REF(SymbolTableRef, mc:symbol_table)
+CASV1_SIMPLE_GROUP_REF(DIEAbbrevSetRef, mc:debug_DIE_abbrev_set)
```

Comment:
Not quite! The point here is so that we can replace CAS references with indices into the the list of references of this block, thus saving 7 bytes per reference

Reply:
I see, that makes sense!